modules/gdscript/gdscript_analyzer.cpp

```diff
@@ -542,6 +542,9 @@ void GDScriptAnalyzer::resolve_class_interface(GDScriptParser::ClassNode *p_clas
 					if (datatype.type_source != GDScriptParser::DataType::UNDETECTED) {
 						datatype.type_source = GDScriptParser::DataType::INFERRED;
 					}
+					if (member.variable->initializer->type == GDScriptParser::Node::GET_NODE && !member.variable->onready) {
+						push_error(vformat(R"*(Use "@onready var %s = get_node(...)" instead.)*", member.variable->identifier->name), member.variable->initializer);
```

Comment:
We should make the message more descriptive so people understand *why* they have to use `@onready`:

```suggestion
						push_error(vformat(R"*(Cannot get node when the scene tree isn't ready. Use "@onready var %s = get_node(...)" instead.)*", member.variable->identifier->name), member.variable->initializer);
```

Reply:
@Calinou added the change.